### PR TITLE
Add a mention of the Reference Architecture to the website

### PR DESCRIPTION
**Pull request by lholmquist**
This adds a link and mentions the Reference Architecture on the website.

I combined the starter and examples section with the reference arch stuff,  which seemed to flow better than adding a whole new section on the landing page.

i also added a little bit of text in the "What is Nodeshift" section

For those who don't want to go through building the site themselves and worrying about ruby version :)  i've included a screenshot of the change to the starter/example section:

Screenshot from 2020-10-22 13-02-04

### CODE REVIEW

File: _includes/home/homepage-startersexamples-band.html [hunk 1]
```diff
@@ -3,8 +3,10 @@
   <div class="highlights-alternating-inline-block">
     <div class="grid-wrapper">
       <div class="width-8-12 width-12-12-m block-content mobile-flip">
-        <h2>STARTERS & EXAMPLES</h2>
-        <p>The Nodeshift project contains a wide range of starter applications showcasing the most basic cloud native concepts, like REST microservices and Health Checks, to more advanced concepts like Cloud Native Security.</p>
+        <h2>Reference Architecture & Examples</h2>
+        <p>The goal of the Node.js Reference Architecture is to present the Nodeshift teams opinion on what components to use when building Node.js applications and guidance on how to be succesful in a production environment.</p>
```
Comment (helio-frota) [inline _includes/home/homepage-startersexamples-band.html]: nit `succesful` `successful`

File: _includes/home/homepage-whatis-band.html [hunk 2]
```diff
@@ -7,7 +7,7 @@ <h2>WHAT IS NODESHIFT?</h2>
     <h3>One of the core concepts of the Nodeshift project, is that your tools shouldn't get in the way. We provided a set of opinionated developer resources for developing Cloud Native Applications on Kubernetes and Openshift.</h3>
```
Comment (helio-frota) [inline _includes/home/homepage-whatis-band.html]: nit `Openshift` `OpenShift`

**Review by helio-frota — CHANGES_REQUESTED** [review]

File: _includes/home/homepage-startersexamples-band.html (hunk 1 above)
Comment (mhdawson) [inline _includes/home/homepage-startersexamples-band.html]:
```suggestion
        <p>The goal of the Node.js Reference Architecture is to present the Red hat and IBM teams opinion on what components to use when building Node.js applications and guidance on how to be succesful in a production environment.</p>
```

Comment (mhdawson) [reply _includes/home/homepage-startersexamples-band.html]: I think this should be a bit broader than the "Nodeshift" team as it is meant to reflect the broader opinion.

Comment (lholmquist) [reply _includes/home/homepage-startersexamples-band.html]: that sounds reasonable

**Comment by lholmquist** [comment]
Updated @mhdawson @helio-frota PTAL

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by helio-frota — APPROVED** [review]

**Pull request merged**